packages/client/src/components/ClientApp.svelte

```diff
@@ -103,140 +120,140 @@
   {/if}
 </svelte:head>
 
-{#if dataLoaded}
-  <div
-    id="spectrum-root"
-    lang="en"
-    dir="ltr"
-    class="spectrum spectrum--medium {$themeStore.baseTheme} {$themeStore.theme}"
-    class:builder={$builderStore.inBuilder}
-  >
-    <DeviceBindingsProvider>
-      <UserBindingsProvider>
-        <StateBindingsProvider>
-          <RowSelectionProvider>
-            <QueryParamsProvider>
-              <!-- Settings bar can be rendered outside of device preview -->
-              <!-- Key block needs to be outside the if statement or it breaks -->
-              {#key $builderStore.selectedComponentId}
-                {#if $builderStore.inBuilder}
-                  <SettingsBar />
-                {/if}
-              {/key}
-
-              <!-- Clip boundary for selection indicators -->
-              <div
-                id="clip-root"
-                class:preview={$builderStore.inBuilder}
-                class:tablet-preview={$builderStore.previewDevice === "tablet"}
-                class:mobile-preview={$builderStore.previewDevice === "mobile"}
-              >
-                <!-- Actual app -->
-                <div id="app-root">
-                  {#if showDevTools}
-                    <DevToolsHeader />
-                  {/if}
+<div
```

Comment:
Previously this wasn't displayed at all until `dataLoaded` was set to true, then the entire tree was rendered at once, which caused a few milliseconds of choppy visuals. Always rendering the tree in a hidden div seems to fix this, and doesn't through any errors, so I assume its okay 🤷

Reply:
Hmm I'm a little apprehensive about this change... But if you're not getting any errors then maybe we're ok. With changes like this I always just worry that there's something written to expect data to be available on the initial load, which now there might not be. If everything is perfectly reactive then this won't be a problem, but it's just a risky change.

If it adds a decent amount of value then I'm happy to risk it.

We couldn't try a (less bulletproof) solution of waiting a few milliseconds before removing the skeleton, and keeping this loading blocker, could we? Tree render speed will vary of course but if we could try some small timeout values and seeing if that also fixes the choppy visuals without any risk.